dbt/adapters/snowflake/connections.py

```diff
@@ -484,6 +487,34 @@ def add_query(
 
         return connection, cursor  # type: ignore
 
+    def _safe_add_standard_queries(
+            self,
+            stripped_queries: List[str],
+            auto_begin: bool = True,
+            bindings: Optional[Any] = None,
+            abridge_sql_log: bool = False,
+    ) -> Tuple[Connection, Any]:  # type: ignore
+        try:
+            connection, cursor = self._add_standard_queries(
+                stripped_queries,
+                auto_begin=auto_begin,
+                bindings=bindings,
+                abridge_sql_log=abridge_sql_log,
+            )
+            return connection, cursor
+        except Exception as e:
+            # Seen happening with dbt.exceptions.DbtDatabaseError exception type:
+            if "already exists" in repr(e):
+                connection, cursor = self._add_begin_commit_only_queries(
```

Comment:
I don't know why some queries run twice now, it is a bit worrying, so this happens when you try to create seed table twice.
'commit' is just a random query that always succeed and allows me to get a cursor, but you are right, it might have adverse effects.